cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -281,7 +282,9 @@ func PerformStaticPodUpgrade(client clientset.Interface, waiter apiclient.Waiter
 		return err
 	}
 
-	return upgrade.StaticPodControlPlane(waiter, pathManager, internalcfg, etcdUpgrade)
+	// These are uninitialized because passing in the clients allow for mocking the client during testing
+	var oldEtcdClient, newEtdClient etcdutil.Client
```

Comment:
This is weird and would cause static analysis issues.

Reply:
We could pass the client creation code as functions?
WDYT?